fmatch/matcher.py

```diff
@@ -230,8 +230,7 @@ def get_agg_metric_query(
         metric_queries = []
         not_queries = [
             ~Q("match", **{not_item_key: not_item_value})
-            for not_item in metrics.get("not", [])
```

Comment:
I think the previous logic was meant to take a list of values under `not` as shown below
```
[{'labels.container.keyword': 'kube-rbac-proxy'}, {'labels.namespace.keyword': 'xyz'}]
``` 
and then generate a query as below
```
[Bool(must_not=[Match(labels__container__keyword='kube-rbac-proxy')]), Bool(must_not=[Match(labels__namespace__keyword='xyz')])]
```

So the yaml structure should be 
```
      not:
        - labels.container.keyword: kube-rbac-proxy
        - labels.namespace.keyword: xyz
```